src/screens/Services/Services.js

```diff
@@ -142,6 +133,46 @@ class ServicesScreen extends React.Component<Props> {
     ];
   };
 
+  getBuyCryptoServices = () => {
+    const {
+      user: { email = '' },
```

Comment:
Is it better to put `null` instead of an empty string? Whilst `''` is falsey, some tools or services might result in this being _something_, instead of nothing...